### PR TITLE
corrected az cli instructions for nixos on azure

### PR DESCRIPTION
**Pull request by ghuntley**
Here's the correct CLI instructions on how to create a nixos virtual machine on Azure. The vhd v16.03 on the homepage does not work but this is being resolved at https://github.com/NixOS/nixpkgs/issues/36262. This pull-request can and should be merged before then.

### CODE REVIEW

**Comment by ghuntley** [comment]
This PR replaces https://github.com/NixOS/nixos-homepage/pull/226 which should be closed. The `azure` command has been deprecated.

File: nixos/download.tt [hunk 1]
```diff
@@ -135,44 +135,74 @@ of NixOS  using azure-cli, just run:</p>
...
-$ azure group create $GROUP --location $LOCATION
+az group create \
+    --name "$RESOURCE_GROUP" \
+    --location "$LOCATION
 
 # create storage
-$ azure storage account create $STORAGE -g $GROUP -l $LOCATION --type LRS
-$ azure storage account keys list $STORAGE -g $GROUP
+az storage account create \
+    --name "$STORAGE_ACCOUNT" \
```
Comment (grahamc) [inline nixos/download.tt]: It looks like STORAGE_ACCOUNT is unset prior to here

Comment (ghuntley) [reply nixos/download.tt]: Resolved and updated.

**Comment by grahamc** [comment]
Thank you!

**Comment by toraritte** [comment]
The Azure and GCE tabs have already been removed in 4ebcceff300104799ee65a86657570fb26e135a7 on 4/14/2020...

**Pull request merged**